File: example/javalib/android/2-linting/1-basic-linting/app/src/main/java/com/helloworld/app/MainActivity.java
package com.helloworld.app;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
import android.view.ViewGroup.LayoutParams;
import android.view.Gravity;


public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // Create a new TextView
        TextView textView = new TextView(this);

        // Set the text to "Hello, World!"
        textView.setText("Hello, World!");

        // Set text size
        textView.setTextSize(32);

        // Center the text within the view
        textView.setGravity(Gravity.CENTER);

        // Set layout parameters (width and height)
        textView.setLayoutParams(new LayoutParams(
            LayoutParams.MATCH_PARENT,
            LayoutParams.MATCH_PARENT));

        // Set the content view to display the TextView
        setContentView(textView);
    }
}

File: libs/init/gradle/test/resources/android-hello-kotlin-gradle8/app/src/main/kotlin/com/helloworld/gradle8/MainActivity.kt
package com.helloworld.gradle8

import android.app.Activity
import android.os.Bundle
import android.view.Gravity
import android.view.ViewGroup.LayoutParams
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val textView = TextView(this)
        textView.text = getString(R.string.app_name)
        textView.textSize = SampleLogic.textSize()
        textView.gravity = Gravity.CENTER
        textView.layoutParams = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT)
        textView.setTextColor(getColor(R.color.text_green))
        textView.setBackgroundColor(getColor(R.color.white))
        setContentView(textView)
    }
}
